dracut-version.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+# shellcheck disable=SC2034
+DRACUT_VERSION=059
```

Comment:
How about hard-code `DRACUT_VERSION` in `dracut.sh`? This would save more lines of code and prevents mismatching `dracut.sh` and `dracut-version.sh`.

Reply:
If it can be overridden if `dracut-version.sh` exists, that looks good by me. But I wouldn't change how `dracut-version.sh` can be built, it's useful for some distros out there.

Reply:
Let's talk code. Here is the proposal: https://github.com/dracutdevs/dracut/pull/2500

Reply:
See https://github.com/dracutdevs/dracut/pull/2500#pullrequestreview-1594996990